### PR TITLE
Correct API docs path and fix pytest invocation

### PR DESCRIPTION
**Pull request by Hritik14**
The earlier location http://127.0.0.1:8000/api/docs was a 404. As the
urls.py suggests, the README now reflects http://127.0.0.1:8000/api/schema/swagger-ui/

After the first install, activating the venv is done prior to installing
pytest, thus the system default pytest is being called. It is either
required to reactive venv or run pytest via python -m.

Signed-off-by: Hritik Vijay <hritikxx8@gmail.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -206,7 +206,7 @@ Run Tests
 Use these commands to run code style checks and the test suite::
 
     black -l 100 --check .
-    DJANGO_DEV=1 pytest
+    DJANGO_DEV=1 python -m pytest
```
Comment (pombredanne) [inline README.rst:209]: If you are in an activate virtualenv, pytest should work. Did you experience an issue here?

Comment (Hritik14) [reply README.rst:209]: Yes. As mentioned [here](https://stackoverflow.com/questions/35045038/how-do-i-use-pytest-with-virtualenv), it fails on the first run. I've generated an [asciinema](https://asciinema.org/a/k7gQuPDlDxSU3iPCl8k3sVEi7) to reproduce the issue as well.

**Comment by sbs2001** [comment]
The pytest bug is weird. @Hritik14  could you try updating the venv pytest to latest and try running `python -m pytest` . Just to determine if issue is caused due  to system pytest invocation or just some version incompability.

**Comment by Hritik14** [comment]
@sbs2001 I think you're right. It is failing because the tests are only compatible with pytest version shipped with the project requirements. 
As you can see, after first installation, `which pytest` points to the system pytest.
I have `pytest 6.2.2` installed and the project requires `pytest==5.3.2`.    
So, it is both due to system pytest invocation and version incompatibility.

Maybe we can look into this in #348.

In any way, for now, I think the pytest invocation should always be the one shipped with the project itself so as to not _just break_ due to any system update.

**Comment by sbs2001** [comment]
@Hritik14  thanks for confirming. Do you mind repasting your comment at https://github.com/nexB/vulnerablecode/issues/348, that'll be handy.

We'll use `python -m pytest` as you suggest. It probably doesn't matter much, but I want this to be consistent with the CI. You'll need to update to https://github.com/nexB/vulnerablecode/blob/ff08fd6044110bbd0b7b57010e3622b2fb25c4b6/.travis.yml#L21  and https://github.com/nexB/vulnerablecode/blob/ff08fd6044110bbd0b7b57010e3622b2fb25c4b6/.github/workflows/main.yml#L41

As for the api doc related changes. That's a good check. IMHO the docs should be at http://127.0.0.1:8000/api/docs, the `urls.py` need some changes.

**Comment by Hritik14** [comment]
@sbs2001 Your wish is my command. :)

**Review by sbs2001 — APPROVED** [review]
@Hritik14  thanks for the PR, I'm merging this

**Pull request merged**